### PR TITLE
Publish minutes of 2021-12-09 meeting

### PR DESCRIPTION
**Pull request by Rob--W**
Generated from https://docs.google.com/document/d/1QkwhEMtMS67JBUkl_WVPZ4lRSKoWcQNlLJSf_GwSXg8/edit using the tool and process from #105.

During this meeting we discussed or mentioned #134, #120, #93 and #118. Although added to the agenda, we did not have enough time to discuss #113 and #115.

### CODE REVIEW

**Comment by nifgraup** [comment]
Was #133 not on the agenda?

**Comment by Rob--W** [comment]
> Was #133 not on the agenda?

Apparently not. But we didn't have enough time to get through all points that were on the agenda.

But #133 is about functionality that's lost if background pages were to be replaced with a service worker with a limited lifetime.
That broader topic was covered extensively during the meeting, as part of issue #134, #120 and the "Random Q&A related to MV3".

Note: if you'd like to discuss the specifics of #133, just post in #133; it's not necessary to wait for the video meetings to participate in the discussion.

File: _minutes/2021-12-09-wecg.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+[Issue 118](https://github.com/w3c/webextensions/issues/118): expose more mv3 APIs to mv2
+
+ * [timothy] In Safari we do expose scripting in MV2. Anything else besides scripting?
+ * [carlos] Another API is the ability to find out whether an action button is pinned on the toolbar. It's not available in MV2, only in MV3, but useful.
+ * [devlin] From the Google side we'd like to support new APIs in MV3, because MV2 is deprecated and supporting new APIs in MV2 adds to the maintenance burden. It's also an incentive for developers to migrate to MV3. We'll fix high-priority bugs in MV2, but MV3 is priority.
+ * [tomislav] Disagree with viewpoint, it makes transitioning harder. Our plans are to support APIs in MV2 where possible.
+ * [timothy] Safari also has that philosophy. We'll also support service workers in MV2.
+ * [devlin] For this reason we support service workers and DNR in MV2 in Chrome. Some other APIs have a less drastic migration and are more utilities than dramatic replacement features, e.g. the action API replaces the browserAction API and is not supported in MV2 because migration is straightforward.
+ * [rob] Firefox develops new features restricted to MV3,
```
Comment (dotproto) [inline _minutes/2021-12-09-wecg.md]: @Rob--W, looks like we only got the first half of your thought. Do you recall what the second half was?

File: _minutes/2021-12-09-wecg.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+   * [devlin] Yes.
+ * [carlos] Does Google consider other chromium browsers with respect to the Manifest V3 timeline?
+   * [devlin] Very much want to be an open source browsers that others can fork and modify, but must also consider our own browser. Want to make sure that we're doing what's best for Chrome without negatively impacting 3rd parties. Won't intentionally make it hard for other vendors.
+ * [glazou] Suggest Google hold an open event with extension vendors to discuss Manifest V3 issues, strategy and timeline
+   * [rob] File issues in the WECG (or issue trackers for vendor-specific bugs)
+
+[Issue 93](https://github.com/w3c/webextensions/issues/93): Support i18n.getMessage API in serviceWorker
+
+ * [carlos] Simeon mentioned in the past that there are issues with this method that may require design changes. Can you clarify?
+ * [devlin] An issue with the implementation is that it's synchronous. [spoke to Chrome implementation details related to process responsibility and message passing]
```
Comment (dotproto) [inline _minutes/2021-12-09-wecg.md]:
```suggestion
 * [devlin] An issue with the implementation is that it's synchronous. [spoke to Chrome implementation details related to process responsibility, message passing between processes, challenges in supporting synchronous responses, and use cases that currently require synchronous responses]
```

I don't recall Devlin's comment with much specific detail, but I think this captures the essence of what he said a little more accurately.

**Review by dotproto — CHANGES_REQUESTED** [review]

**Comment by Rob--W** [comment]
Thanks for the review @dotproto! I have updated the PR, the diff is at https://github.com/w3c/webextensions/compare/add3df59d4e9fd5c9f6cd2f454298319ad63a5f9..445963dd60096b64da4adf571b2d29b5961e970a

**Review by dotproto — APPROVED** [review]
Thanks for the updates, @Rob--W. I think we're good to go :)

**Pull request merged**